internal/provider/resource_gitlab_project_access_token.go

```diff
@@ -90,17 +90,27 @@ var _ = registerResource("gitlab_project_access_token", func() *schema.Resource
 				Type:        schema.TypeInt,
 				Computed:    true,
 			},
+			"access_level": {
+				Description:      fmt.Sprintf("The access level of the associated user_id project membership (cf resource_gitlab_project_membership). Valid values are: %s. Default is `%s`", renderValueListForDocs(validProjectAccessLevelNames), accessLevelValueToName[gitlab.MaintainerPermissions]),
+				Type:             schema.TypeString,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice(validProjectAccessLevelNames, false)),
```

Comment:
@MrLuje  - We have examples of both in the code base, but is there a reason in this specific case why we're ignoring casing on the input string?